### PR TITLE
[qt] Make network disabled icon 50% opaque

### PR DESCRIPTION
**Pull request by maflcko**
The png was sampled too low, so displayed incorrectly on high res displays. Also, I modified the alpha channel a bit.

### CODE REVIEW

File: src/qt/res/src/network_disabled.svg [hunk 1]
```diff
@@ -5,11 +5,11 @@
    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
    xmlns:svg="http://www.w3.org/2000/svg"
    xmlns="http://www.w3.org/2000/svg"
-   id="svg2"
```
Comment (paveljanik) [inline src/qt/res/src/network_disabled.svg]: isn't this only re-sorted?

Comment (MarcoFalke) [inline src/qt/res/src/network_disabled.svg]: Inkscape did this.

Comment (luke-jr) [inline src/qt/res/src/network_disabled.svg]: So don't use Inkscape...?

File: src/qt/res/src/network_disabled.svg [hunk 2]
```diff
@@ -5,11 +5,11 @@
...
-   viewBox="0 0 24 24"
-   height="24"
+   version="1.2"
    width="24"
-   version="1.2">
+   height="24"
```
Comment (luke-jr) [inline src/qt/res/src/network_disabled.svg]: The size of the SVG isn't changed...

**Comment by jonasschnelli** [comment]
utACK fa30b060eec1f51238292a63039dcc7943abf236

SVG's XML is interpreted and created differently. No need for bikeshedding there.

**Comment by Victorsueca** [comment]
ACK fa30b06
Tested on Windows x64
GUI Screenshot

File: src/qt/res/src/network_disabled.svg (hunk 1 above)
Comment (MarcoFalke) [inline src/qt/res/src/network_disabled.svg]: Fixed sorting nit

File: src/qt/res/src/network_disabled.svg (hunk 2 above)
Comment (MarcoFalke) [inline src/qt/res/src/network_disabled.svg]: This is not required. You can tell the program of your choice what dpi to sample at.

Nonetheless, I changed it to `92`, as well as for all `connect-$N.svg`

**Comment by paveljanik** [comment]
Nice and reviewable now 👍 
ACK https://github.com/bitcoin/bitcoin/pull/9145/commits/fa42e4a082e409794d0a00a8bbffc35142a17d32

**Comment by paveljanik** [comment]
Thanks to both Marco and Luke!

**Pull request merged**